main.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright (C) 2024 Nordix Foundation.
+For a full list of individual contributors, please see the commit history.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/cdevents/webhook-cdevents-adapter/pkg/cdevents"
+	"github.com/hashicorp/go-plugin"
+)
+
+func run() error {
+	translator, err := cdevents.LoadConfig("translator-plugins.yaml")
+	if err != nil {
+		log.Fatalf("Error loading translator plugins Config: %s\n", err)
+		return err
+	}
+	log.Printf("Loaded translator plugins Config: %s\n", translator)
+
+	port := 8080
+	log.Printf("### Starting CDEvents Webhook Adapter ###\n")
+	log.Printf("Server listening on :%d\n", port)
+	for _, translatorPlugin := range translator.Translator.Plugins {
```

Comment:
The logic to download plugins could eventually move into a dedicated testable function